src/components/admin/form-components/withValidate.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import { getDisplayName } from 'lib/higher-order-helpers';
+
+const withValidate = (...validators) => (WrappedField) => {
+  const ValidatedField = (originalProps) => {
+    const failedValidator = validators
+      .find(validator =>
+        validator(originalProps.value)
+      );
+    let ReturnedComponent;
+    if (failedValidator) {
+      ReturnedComponent =
+        (<WrappedField
+          errorText={failedValidator(originalProps.value)}
+          {...originalProps}
+        />);
+    } else {
+      ReturnedComponent = (<WrappedField {...originalProps} />);
```

Comment:
No need for the parenthesis on a single line here

Reply:
👍 - TODO